apps/meteor/client/views/teams/contextualBar/channels/RoomActions.js

```diff
@@ -81,10 +81,25 @@ const RoomActions = ({ room, reload }) => {
 	const menuOptions = useMemo(() => {
 		const AutoJoinAction = async () => {
 			try {
-				await updateRoomEndpoint({
+				const { numberOfMembers } = await updateRoomEndpoint({
 					roomId: rid,
 					isDefault: !room.teamDefault,
 				});
+
+				const MAX_NUMBER_OF_AUTO_JOIN_MEMBERS = 1000;
```

Comment:
Is there a constants file I could put this one?

Reply:
Yes, we can share the same constant on client and server by putting it in some `meteor/lib` folder context.

Any code inside `lib` folder is available on both client and server, for example take a look at - `apps/meteor/lib/videoConference/constants.ts` file